gfx/Larwick_Overmap/pngs_overmap_16x16/dirt_road/dirt_road.json

```diff
@@ -1,4 +1,59 @@
 [
+  {
+    "id": [
+      "ants",
+      "ants_acid"
```

Comment:
Dirt roads for anthills?

Reply:
This only affects their tunnels underground.